listening-comp/backend/vector_db.py

```
@@ -0,0 +1,568 @@
+import os
+import sqlite3
+import numpy as np
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)